main.tf

```diff
@@ -35,7 +37,7 @@ resource "aws_vpc_endpoint" "this" {
   count = module.this.enabled ? 1 : 0
 
   vpc_id              = var.vpc_id
-  service_name        = data.snowflake_system_get_privatelink_config.this.aws_vpce_id
+  service_name        = data.snowflake_system_get_privatelink_config.this[*].aws_vpce_id
```

Comment:
don't we need `one(..)` here? 🤔

Reply:
Yeah, I forget about this one 6eaff004f7eab1ad4111e76d205032193a74bb87